lab1.js

```diff
@@ -71,8 +81,31 @@ var sentence2 = 'Come over here so you can scratch my belly.';
  **two** different kinds of loops to implement this. (10 points each.)
  HINT: the 'split' method on String will be useful.
 */
+/*function assert(sentence,split) {
+  	var res=sentence.split("")
+  }
+}*/
+	var sentence1 = 'More food please.';
+var res=sentence1.split(" ");
+sentence1="";
+for(var i=1;i<res.length;i++)
+{
+	sentence1= sentence1+" chirp "
+
+}
+sentence.trim();
+sentence1= sentence1+" chirp. "
 
 //your code goes here
+var sentence2 = 'Come over here so you can scratch my belly.';
+var reso=sentence2.split(" ");
+sentence2=" ";
+for(var i=1;i<reso.length;i++)
+{
+	sentence2=sentence2+ " chirp ";
+}
+sentence2.trim();
+sentence2=sentence2+" chirp."
 
 assert(sentence1 === 'chirp chirp chirp.', 'sentence 1 should have 3 chirps');
```

Comment:
your asserts are failing because you have double the spaces you need. like this:
" chirp  chirp  chirp. "
instead of 
"chirp chirp chirp."